rpc_component/schemata.py

```diff
@@ -134,20 +135,59 @@ def is_version_ids_unique(releases):
     }
 )
 
-component_metadata_schema = Schema(
+dependencies_schema = Schema(
     {
-        "dependencies": And(
+        Optional("dependencies"): And(
             [
                 {
                     "name": And(str, len),
                     "constraints": constraints_schema,
                 },
             ],
             is_value_unique("name"),
+        )
+    }
+)
+
+artifacts_file_schema = Schema(
+    {
+        "type": "file",
+        "source": And(str, len),
+        Optional("dest"): And(str, len),
+        Optional("expire_after"): And(str, len)
+    }
+)
+
+artifacts_log_schema = Schema(
+    {
+        "type": "log",
+        "source": And(str, len)
+    }
+)
+
+artifacts_schema = Schema(
+    {
+        Optional("artifacts"): And(
```

Comment:
I don't think you need the `And` because there is only the list.

Reply:
👍